common/github.com/IBM/vpc-go-sdk/vpcv1/vpc_v1.go

```diff
@@ -122,7 +122,7 @@ func NewVpcV1(options *VpcV1Options) (service *VpcV1, err error) {
 	}
 
 	if options.Version == nil {
-		options.Version = core.StringPtr("2023-08-08")
+		options.Version = core.StringPtr("2023-09-01")
```

Comment:
Since this is a breaking change the version should be `2024-01-01` to test backup creation using versioned api change else it will be un-versioned api support ( Just for backup apis )

Reply:
You might need to add future_params for backup apis.